website/versioned_docs/version-v3.9.x/gator.md

```diff
@@ -225,12 +237,51 @@ gator verify path/to/suites/... --run "disallowed"
 
 Run `gator verify --help` for more information.
 
+## The `gator expand` subcommand
+
```

Comment:
can we clarify this will be available in GK v3.10+

Reply:
Should we only add these changes to the unversioned docs? That should make this implicit:

https://github.com/open-policy-agent/gatekeeper/tree/master/website/docs

In general, these changes should be in unversioned docs anyway, but I think they should *only* be in unversioned docs.

Reply:
Moved to unversioned docs.